Bucketeer/Sources/Internal/Remote/ApiClientImpl.swift

```diff
@@ -114,6 +115,11 @@ final class ApiClientImpl: ApiClient {
         timeoutMillis: Int64,
         encoder: JSONEncoder = JSONEncoder(),
         completion: ((Result<(Response, URLResponse), Error>) -> Void)?) {
+        if (closed) {
+            completion?(.failure(BKTError.illegalState(message: "API Client has been closed")))
```

Comment:
Just confirming. Will this be reported via error metric events? If so, what metric will be used?

Reply:
Yes, It will. 
It will be tracked as `MetricsEventType.internalError`

Reply:
https://github.com/bucketeer-io/ios-client-sdk/blob/196c2c98501f5bb54548d7b9a71bf0fdf5c5fd38/Bucketeer/Sources/Internal/Event/EventInteractor.swift#L374